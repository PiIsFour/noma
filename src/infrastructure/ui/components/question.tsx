import React from 'react'

interface Props {
	kanji: string
}

export const Question = ({ kanji }: Props): JSX.Element => {
	return <div>
		{kanji}
	</div>
}
