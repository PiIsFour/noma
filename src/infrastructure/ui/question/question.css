.question {
	color: snow;
	font-size: calc(10px + 2vmin);
	letter-spacing: .1rem;
}