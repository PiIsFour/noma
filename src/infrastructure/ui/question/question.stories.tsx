import React from 'react'
import { ComponentStory, ComponentMeta } from '@storybook/react'
import { Question } from './question'

export default {
	title: 'Scenes/Question',
	component: Question,
	argTypes: {
	},
} as ComponentMeta<typeof Question>

const Template: ComponentStory<typeof Question> = (args) => <Question {...args} />

export const Standard = Template.bind({})
Standard.args = {
	kanji: 'さくらが歩く',
}
