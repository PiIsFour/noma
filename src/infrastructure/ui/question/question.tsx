import React from 'react'
import './question.css'

interface Props {
	kanji: string
}

export const Question = ({ kanji }: Props): JSX.Element => {
	return <div className="question">
		{kanji}
	</div>
}
