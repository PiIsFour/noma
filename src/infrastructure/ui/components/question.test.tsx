import React from 'react'
import { render, screen } from '@testing-library/react'
import { shallow } from 'enzyme'

import { Question } from './question'

describe('Question', () => {
	it('matches snapshot', () => {
		const wrapper = shallow(<Question kanji='さくらが歩く。' />)
		expect(wrapper).toMatchSnapshot()
	})
})
