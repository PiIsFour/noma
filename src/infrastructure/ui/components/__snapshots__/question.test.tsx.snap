// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question matches snapshot 1`] = `
<div>
  さくらが歩く。
</div>
`;
